modules/seqexec/server/src/main/scala/seqexec/server/gpi/GpiInstrumentActions.scala

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2016-2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec.server.gpi
+
+import cats._
+import cats.implicits._
+import fs2.Stream
+import io.chrisdavenport.log4cats.Logger
+import seqexec.engine.ParallelActions
+import seqexec.engine.Result
+import seqexec.server.StepType
+import seqexec.server.InstrumentActions
+import seqexec.server.ObserveActions
+import seqexec.server.ObserveEnvironment
+
+/**
+  * Gpi needs different actions for A&C
+  */
+class GpiInstrumentActions[F[_]: MonadError[?[_], Throwable]: Logger]
```

Comment:
Gpi also produces custom Actions for A&C but it is much simpler, we just don't do anything